fala/assets-src/scripts/find-legal-adviser.js

```diff
@@ -2,6 +2,28 @@ var $ = require('jquery');
 var debounce = require('lodash/function/debounce');
 var find = require('lodash/collection/find');
 var reduce = require('lodash/collection/reduce');
+const wideScreen = 641;
+const originalHeight = 465;
+
+function setMapHeight(setHeight) {
+  $('body, html').css('overflow', 'hidden'); //remove scroll bar
+  let screenWidth = $(window).width();  //get width as per CSS
+  $('body, html').css('overflow', 'visible'); //restore scroll bar
+  if (screenWidth >= wideScreen) {
+    let listHeight = $("ul.org-list").height();
+    let paginationHeight = $("nav.search-results-pagination").height() + 6 + 2; /*mimicking the CSS*/
+    if (listHeight > setHeight + paginationHeight) setHeight = listHeight;
+    $(".search-results-list").height(setHeight);
+    $(".search-results-container").height(setHeight + paginationHeight);
+  } else {
+    $(".search-results-list, .search-results-container").height("");
+  }
```

Comment:
Is this necessary as shouldn't the media query changes fix this? Could we not remove the else statement for the media query should kick in when the if statement fails?

Reply:
I agree